packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.stories.js

```diff
@@ -81,6 +87,9 @@ storiesOf("Organisms|GroupedProduct", module)
     props: {
       settings: {
         default: object("settings", { type: "slider" }, "Props")
+      },
+      disabled: {
```

Comment:
@aniamusial it's not good name :) what do you think about hasCarousel: { default: true }